data/test/scenarios/manual_tests/overlay_text_demo.cfg

```diff
@@ -1,6 +1,6 @@
 #textdomain wesnoth-test
 
-[test]
+[multiplayer]
```

Comment:
Why do some `[test]`s get changed to be `[multiplayer]`?

Reply:
For consistency with all scenarios in manual_tests.

Reply:
That's not an answer. Why are they changed? The point of a `[test]` scenario is that you can launch it via the `-t` command-line option or the "test list" from the main menu. If you change it to a `[multiplayer]` scenario, then that won't work, right?

`[test]` does not mean it's a unit test, after all.

Unless you have some strong reasoning, I think it's incorrect to change these.

Reply:
It makes them all available via the in-game UI when launching Wesnoth with the -d argument.

Reply:
To be honest I'm not really sure why we currently have some hidden behind the -d flag and some hidden behind the -t flag. If nothing else, test scenarios that require manual playing to use should be handled consistently.

Reply:
It makes them available via the "create a multiplayer game" UI, allowing the user to select an era and requiring the user to select side controllers and leader types in the GUI. This doesn't seem good.

Reply:
It breaks the (useful when working on the command line) ability to launch those tests directly with `wesnoth -t testname`.

It's made `wesnoth -t` (without a testname) show an `Error: unknown scenario` instead of launching the dumping-ground test scenario. (That I'm calling it the "dumping ground" is something that would need a separate PR to fix.)

Reply:
Changed back.